src/big5prober.js

```diff
@@ -27,16 +27,19 @@
  * 02110-1301  USA
  */
 
-!function(jschardet) {
+var CodingStateMachine = require('./codingstatemachine')
```

Comment:
We can also use this reorganization to make the filenames camel case, so they match their symbols: `CodingStateMachine.js`.